docsource/developers.rst

```diff
@@ -22,9 +22,13 @@ Branches:
 
 Recompiling the generated code
 ------------------------------
-To compile, simply run ``make compile``. The
-`kb-sdk <https://github.com/kbase/kb_sdk>`_ executable must be in the system
-path.
+To compile, run ``./gradlew compile``.
+
+Note that:
+
+* The `kb-sdk <https://github.com/kbase/kb_sdk>`_ executable must be in the system path.
+* The WorkspaceServer class compiled in constructor has been commented out in order to use
+  a custom constructor. That must be reversed for the compile to work.
```

Comment:
"That code must be reinstated"? "reversed" doesn't quite sound right

Reply:
Clarified, is that better?